### ISSUE TITLE
Incorporate Axe

### ISSUE THREAD

**Issue by 13protons** [13protons/empath#2]
Axe will evaluate a website's accessibility. This should be injected into the <webview> preload, and run on any change to `window.location`

https://www.npmjs.com/package/axe-core
